tests/detection/test_map.py

```diff
@@ -59,7 +55,7 @@
             ),  # coco image id 74
             dict(
                 boxes=torch.Tensor([[0.00, 2.87, 601.00, 421.52]]),
-                scores=torch.Tensor([0.699, 0.423]),
+                scores=torch.Tensor([0.699]),
```

Comment:
That score it probably a left over from an additional ground truth. It should have been caught by the `_input_validator`, I fixed the issue. For previous calculation it probably didn't matter because it was not returned by any index based loop.